recipes/botor/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "botor" %}
+{% set version = "0.0.1dev8" %}
```

Comment:
Is this an official, or pre-release package? If the latter, you will probably need to use the `conda-forge prerelease` target instead.

Reply:
@npavlovikj this is a pre-release package, available for download from an official source, PyPi.  I think that the target tag must be added to the conda-forge.yml after the feedstock is created, like done here with the "dev" target: https://github.com/conda-forge/sense2vec-feedstock/blob/master/conda-forge.yml

Reply:
You can actually add the `conda-forge.yml` file here, as in https://github.com/conda-forge/staged-recipes/pull/10497/files.

Reply:
Thank you - I added it as in that example you linked